spec/features/accessibility_spec.rb

```diff
@@ -61,4 +61,16 @@
         .according_to(:wcag2a, :wcag2aa, :wcag21a, :wcag21aa)
     end
   end
+
+  context 'when visiting locker applications page' do
```

Comment:
@vh8154 can you group the spec under one describe when you're signed in as an dmin?